INFO     log:set_app.py:325 头像更换成功，用例无异常！